editor.planx.uk/package.json

```diff
@@ -11,9 +11,9 @@
     "@feedback-fish/react": "^1.2.1",
     "@material-ui/core": "^4.12.4",
     "@mui/icons-material": "^5.11.16",
-    "@mui/material": "5.11.9",
-    "@mui/styles": "5.11.9",
-    "@mui/utils": "5.11.9",
+    "@mui/material": "5.12.3",
+    "@mui/styles": "5.12.3",
+    "@mui/utils": "5.12.3",
```

Comment:
This version bump fixed the annoying bug where the "Change" text within a button on the SummaryList wasn't correctly inheriting the font-family.

https://github.com/mui/material-ui/pull/33621

However, upgrading right to the latest version (5.13.1) was causing issues which I spent a lot of time trying to resolve. This version just came out a few days ago so hopefully it's something that will get ironed out shortly.